.github/workflows/unused-deps.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_dispatch: {}
+  # Once per day at 00:00 UTC
+  push:
+    branches: [main]
+  pull_request:
+
+env:
+  RUSTFLAGS: -D warnings
+
+jobs:
+    unused-deps:
+        runs-on: ubuntu-latest
+        name: unused dependencies
+        steps:
+        - name: Checkout sources
+          uses: actions/checkout@v3
+        - name: Install toolchain
+          uses: dtolnay/rust-toolchain@nightly
+
+        - name: Install cargo-udeps
+          run: cargo install cargo-udeps --locked
```

Comment:
Use `taiki-e/cache-cargo-install-action@v1` for this. Check how we install `wasm-pack` in `ci.yml`

Reply:
done